### DISCUSSION TITLE
Get detail statistics about inflight messages

### DISCUSSION THREAD

**Discussion by YraganTron** [emqx/emqx#12849] [Q&A] [answered]
Hi! I found a new feature in the latest release

> Implemented HTTP APIs to get the list of client's in-flight and message queue (mqueue) messages. These APIs facilitate detailed insights and effective control over message queues and in-flight messaging, ensuring efficient message handling and monitoring.

Is this a new feature only available for the latest release? Or in previous versions there is also such functionality, just not for the HTTP API

**Comment by zmstone** [comment, marked as answer]
Hi @YraganTron 
This is releasd in 5.6.0.
It is not ported back to older versions.

**Comment by id** [comment]
@YraganTron Not really. Technically it was possible, but you'd need to do quite a lot of low level calls in emqx erlang shell to get the same results. This patch introduced not just HTTP API, but also new high level internal API to collect and aggregate this data.
